HIP-xxxx.md

```diff
@@ -0,0 +1,81 @@
+# HIP-xxxx : TXT Record Naming Standard for Domain Listings
+
+```
+Number:  HIP-xxxx
+Title:   TXT Record Naming Standard for Domain Listings
+Type:    Standards
+Status:  Draft
+Authors: 0xStefan <https://twitter.com/0xStefan>
+Created: 2022-02-22
+```
+
+## Abstract
+
+A proposal to create a standard for the labels of string attributes in TXT records for domain listings — following the [RFC 1464](https://datatracker.ietf.org/doc/html/rfc1464) standard.
+
+## Motivation
+
+If a domain is not listed on an open marketplace like Namebase or Shakedex, it's hard to know if a domain is for sale by the owner or not. Furthermore a domain seller has no reliable way to provide that information.
+
+## TXT Record Standard: Domain Listing
+
+For the above to work we need standards so any website or app can properly render the TXT records. The records should be flexible, short and easy to understand. It is important to note, that it's impossible to validate the data via HSD. Therefore TXT records must be validated by the site or app that is rendering the TXT records.
+
+The general syntax for domain listings is:
+
+**TXT** `listing <attribute name>=<attribute value>`
+
+There are currently two supported attributes: **[price](#price)** and **[url](#url)**.
+
+### <a id="price"></a> (1) Listing Attribute: Price (mandatory)
+
+The listing price attribute is mandatory to create a domain listing. It enables a domain owner to provide a listing price for the domain. The expected value is the unit as a string, followed by the amount as a number. If the number contains decimals, a dot must be used as a seperator.
+
+```
+TXT listing price=<unit number>
+```
+
+#### Valid Listing Price Examples
+
+```
+TXT listing price=HNS 1000
```

Comment:
What about adding optional symbol `>=` for minimum offer price?

Reply:
That is a great idea! 

Although I think it's better to move the ">" after the equals sign? I know it's unintuitive for devs, but better in regards of the RFC 1464 standard. Don't you think?

Reply:
Sure!

Reply:
#### Note to self and as future reference
The reason why the ">" should be moved AFTER the equals sign is because of the RFC 1464 standard.

Quote:
> The name and value are separated by an equals sign

">" belongs to the value, so it should be after the equals sign. Therefore we can be sure, everything prefixing the equals sign is the name/label of the data.